pom.xml

```diff
@@ -40,8 +40,8 @@
 	</modules>
 	
 	<properties>
-		<openMRSVersion>1.9.4</openMRSVersion>
-		<openMRSBuildVersion>1.9.4</openMRSBuildVersion>
+		<openMRSVersion>2.1.0</openMRSVersion>
```

Comment:
Why are you switching to OpenMRS version 2.1.0? Can't you just use 1.9.9? https://wiki.openmrs.org/display/docs/Supporting+Platform+2.0+and+below

Reply:
i successfully introduced the 1.9.9 version ,i go on with the work in PersonAttributeTypeDeployHandler.java as above where i introduced the DbSessionFactory and DbSession.But on building i get a DbSessionFactory class not found error? could i be missing out some thing in my code

Reply:
have you committed?

Reply:
yes i have https://github.com/openmrs/openmrs-module-metadatadeploy/pull/8/commits/caeebe2b28c058a2fa4c5ddb13d078b16940f2dc

Reply:
yes i have committed!!  https://github.com/openmrs/openmrs-module-metadatadeploy/pull/8/commits/caeebe2b28c058a2fa4c5ddb13d078b16940f2dc

Reply:
If you have committed, then travis should show you the current error log

Reply:
according to travis i get (cannot  find symbol method setPrecise) as seen here https://pastebin.com/WVVnNEvX   now i am trying on another version of core as per the link you dropped me.

Reply:
it still puts out (java.lang.ClassNotFoundException: org.openmrs.api.db.hibernate.DbSessionFactory) yet i successfully import this class!!

Reply:
Is that reported by travis?

Reply:
travis is saying  (cannot find symbol method setPrecise) as seen here  https://pastebin.com/WVVnNEvX

Reply:
i am just wondering could i be missing out some thing during the importaion.I  get  (java.lang.ClassNotFoundException: org.openmrs.api.db.hibernate.DbSessionFactory).Yet on the importation,i successfully import the DbSessionFactory

Reply:
Does this pull request have your changes?

Reply:
yes it does in the .java/org/openmrs/module/metadatadeploy/handler/impl/PersonAttributeTypeDeployHandler.java

Reply:
Then why doesn't travis report the same error?

Reply:
let me commit again!